nrf_modem/doc/ug_nrf_modem_porting_os.rst

```diff
@@ -270,119 +278,222 @@ Reference template for the nrf_modem_os.c file
 The following code snippet shows a simple implementation of the Modem library OS abstraction layer.
 You can use it as a template and customize it for your OS or scheduler.
 
+
 .. code-block:: c
 
-   #include <nrf_modem.h>
-   #include <nrf_modem_os.h>
-   #include <nrf_errno.h>
-   #include <nrf_modem_platform.h>
-   #include <nrf_modem_limits.h>
-
-   #include <nrf.h>
-   #include "errno.h"
-
-    void read_task_create(void)
-	{
-		// The read task is achieved using SW interrupt.
-		NVIC_SetPriority(NRF_MODEM_APPLICATION_IRQ, NRF_MODEM_APPLICATION_IRQ_PRIORITY);
-		NVIC_ClearPendingIRQ(NRF_MODEM_APPLICATION_IRQ);
-		NVIC_EnableIRQ(NRF_MODEM_APPLICATION_IRQ);
-	}
-
-    void nrf_modem_os_init(void)
-    {
-        read_task_create();
-        // Initialize timers / sleeping threads used in the nrf_modem_os_timedwait function.
-        // If enabled, initialize the trace task and mediums.
+    #include <nrf_modem.h>
+    #include <nrf_modem_os.h>
+    #include <nrf_errno.h>
+    #include <nrf.h>
+    #include "errno.h"
+
+
+    /**
+    * @brief Initialize the glue layer and required peripherals
```

Comment:
Comment indentation seem off
```suggestion
    /**
     * @brief Initialize the glue layer and required peripherals
```

Reply:
The API documentation (doxygen) are always generated from the header file. 
c files are internal to the developer and its up to developer to add it as a normal comment or in doxygen format. I think its always the standard comments that I have seen in the c files (I may be wrong).

Reply:
Alright, changed comments to standard comments.